x-pack/plugins/observability/server/lib/rules/threshold/threshold_executor.ts

```diff
@@ -239,16 +238,14 @@ export const createMetricThresholdExecutor = ({
         ? AlertStates.NO_DATA
         : shouldAlertFire
         ? AlertStates.ALERT
-        : shouldAlertWarn
-        ? AlertStates.WARNING
         : AlertStates.OK;
 
       let reason;
-      if (nextState === AlertStates.ALERT || nextState === AlertStates.WARNING) {
+      if (nextState === AlertStates.ALERT) {
         reason = alertResults
           .map((result) =>
             buildFiredAlertReason({
-              ...formatAlertResult(result[group], nextState === AlertStates.WARNING),
```

Comment:
We can remove this boolean parameter in the formatAlertResult function, and remove its usage there as well.